### ISSUE TITLE
Animated stickers are unusable in sidebar

### ISSUE THREAD

**Issue by AuroraWright** [overtake/TelegramSwift#329]
If the sticker sidebar is enabled and you have a few (I have 3) packs of animated stickers, as soon as you open the sidebar they start stuttering and freezing and you can't even preview them, making them unusable. Disabling the sidebar fixes the issue.

**Comment by AuroraWright** [comment]
It seems the CPU can't keep up with rendering all animated stickers at once, maybe as Telegram only uses a single thread? (just a guess). It's also dependent on resolution, a friend told me it doesn't happen on a 1080p screen while I have a retina macbook pro.

**Comment by overtake** [comment]
Yes stickers uses 1 thread. I can add more threads but i'm not sure is good idea because all you desktop resources will be spend for stickers.

**Comment by qmmp123** [comment]
Can you make option to disable animation in side bar and show animation if mouse cursor over?
